poll/poll.go

```
@@ -27,11 +27,13 @@ func PollIncidents(startTime time.Time, light lights.Light, logger *types.Logger
 	notifiedIncidents := make(map[int]bool)
 	seenIncidents := make(map[int]bool)
 	var cachedIncidents []types.Incident
+	currentLightState := "green" // Track current light state
 
 	for {
 		if err := network.CheckConnectivity(); err != nil {
 			logger.WarnLog.Printf("Internet connectivity issue: %s", err.Error())
 			light.On(lights.StateYellow)
```

golangci-lint (1.62.2)

Error return value of `light.On` is not checked

(errcheck)